### ISSUE TITLE
Statusnet: Dents with Pictures looks ugly (Feature Request)

### ISSUE THREAD

**Issue by ghost** [friendica/friendica#1077]
A Dent with a Picture included looks like:
![ugly](https://cloud.githubusercontent.com/assets/5044393/4008948/0d344b6c-29de-11e4-8f66-94c697e570d5.png)

Hopefully that could change to something look better.

**Comment by tobiasd** [comment]
What if we skip the image and just post a link letting GNUsocial decide how to embed the image?

**Comment by annando** [comment]
@tobiasd AFAIK this was an issue with the Statusnet connector and no federation issue.

**Comment by tobiasd** [comment]
The screenshot is from the Vier theme, a posting from a GS contact, so I think the problem is our representation of the posted dent.

**Comment by Hypolite** [comment]
What is a "Dent"? The name of posts in StatusNet?

**Comment by tobiasd** [comment]
initially it was a posting on iDENTica but when Laconica (which later became StatusNet) started to work decentralized it was used for postings in the network in general. Digital campfire stories :-)

**Comment by annando** [comment]
@tobiasd Just to be nitpicking: It is some connector thing. You can detect this on the lock symbol - which appears with the connector but not via OStatus ;-)

Attached images via OStatus are looking different - they are only showed as attachments. We can - and should - work on this as well, but this would be a different issue.

**Comment by tobiasd** [comment]
Ok, then it is an addon issue :-)

**Comment by tobiasd** [comment]
Was this also addressed in #3587?

**Comment by annando** [comment]
No, this is totally unrelated. This here covers only stuff from the Statusnet-Connector, not the OStatus protocol. And so the whole coding would be inside the "addons" section, not inside the core.

But since the OStatus implementation in the core is now really good, I don't have much energy to do anything to this specific addon, I must confess.

**Comment by annando** [comment]
We discontinued both the statusnet addon as well as the OStatus protocol, so I guess we can close this issue.
